WPMedia/fusion-news-theme-blocks
# Tmedia-745:  large manual promo block refactor

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description

- Updated Large Manual promo block to use v2 architecture
- Updated test cases and README

## Jira Ticket

- [TMEDIA-745](https://arcpublishing.atlassian.net/browse/TMEDIA-745)

## Acceptance Criteria

No special notes in Jira ticket. Followed [checklist](https://arcpublishing.atlassian.net/wiki/spaces/TI/pages/3106243821/Converting+a+v1+block+to+v2+architecture) for migrating blocks from v1 to v2.

## Test Steps

- Checkout this branch git checkout TMEDIA-745-large-manual-promo-block-refactor
- Run storybook with npm run storybook and verify functionality corresponding with stories.


## Effect Of Changes

### Before

- Using V1 architecture

<img width="746" alt="Screen Shot 2022-06-30 at 10 35 08 AM" src="https://user-images.githubusercontent.com/83021791/176704775-97cb8c14-5a9b-4cb0-b57a-c8fec253a4e3.png">
<img width="835" alt="Screen Shot 2022-06-30 at 10 35 24 AM" src="https://user-images.githubusercontent.com/83021791/176704782-fe74d33c-8177-432c-8195-f01b78419996.png">


### After

- Migrated to V2 architecture
<img width="1050" alt="Screen Shot 2022-07-05 at 10 26 05 AM" src="https://user-images.githubusercontent.com/83021791/177351273-9658fbdd-69c9-4f26-b3ad-b6f06d48dc51.png">
<img width="406" alt="Screen Shot 2022-07-05 at 10 26 23 AM" src="https://user-images.githubusercontent.com/83021791/177351279-652a4e78-8386-4b52-8da9-729fa7ebf5bf.png">

## Author Checklist

        please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._
